content/en/docs/components/hyperparameter-tuning/resume-experiment.md

```diff
@@ -30,85 +32,101 @@ to make experiment changes. For example, run:
 is attached to the suggestion's deployment.
 
-Katib controller creates [persistent volume claim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) (PVC)
-and [persistent volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistent-volumes) (PV)
-in addition to the suggestion's deployment and service.
+Katib controller creates
+[persistent volume claim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims)
+(PVC) and
+[persistent volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistent-volumes)
+(PV) in addition to the suggestion's deployment and service.
 
-**Note that** If you specify [storage class name](https://kubernetes.io/docs/concepts/storage/storage-classes/)
-in Katib configuration settings for the PVC differently from `katib-suggestion`, controller creates only PVC.
-Follow the [Katib configuration guide](/docs/components/hyperparameter-tuning/katib-config/#suggestion-volume-settings)
-to set up suggestion's volume settings.
+**Note:** If you specify
+[storage class name](https://kubernetes.io/docs/concepts/storage/storage-classes/)
+in Katib configuration settings for the PVC differently from `katib-suggestion`,
+Katib controller creates only PVC. Follow the
+[Katib configuration guide](/docs/components/hyperparameter-tuning/katib-config/#suggestion-volume-settings)
+to set up the suggestion's volume settings.
 
-- PVC is deployed with the name: `<suggestion-name>-<suggestion-algorithm>` in suggestion namespace.
+- PVC is deployed with the name: `<suggestion-name>-<suggestion-algorithm>`
+  in the suggestion namespace.
 
-- PV is deployed with the name: `<suggestion-name>-<suggestion-algorithm>-<suggestion-namespace>`
+- PV is deployed with the name:
+  `<suggestion-name>-<suggestion-algorithm>-<suggestion-namespace>`
 
-After experiment is succeeded, suggestion's deployment and service are deleted.
-Suggestion data can be retained in the volume.
-When you restart the experiment, suggestion's deployment and service are created and
-suggestion statistics can be recovered from the volume.
+After the experiment has succeeded, the suggestion's deployment and
+service are deleted. Suggestion data can be retained in the volume.
+When you restart the experiment, the suggestion's deployment and service
+are created and suggestion statistics can be recovered from the volume.
 
-When you delete the experiment, suggestion's deployment, service, PVC and PV are deleted automatically.
+When you delete the experiment, the suggestion's deployment, service,
+PVC and PV are deleted automatically.
 
-See the [from volume policy example](https://github.com/kubeflow/katib/blob/master/examples/v1beta1/resume-experiment/from-volume-resume.yaml#L18).
+See the
+[from volume policy example](https://github.com/kubeflow/katib/blob/master/examples/v1beta1/resume-experiment/from-volume-resume.yaml#L18).
```

Comment:
for a11y:

```suggestion
Check the
[`from-volume-resume.yaml`](https://github.com/kubeflow/katib/blob/master/examples/v1beta1/resume-experiment/from-volume-resume.yaml#L18)
example to learn more.
```

WDYT? It's more precise, since it's not a "tutorial" example, just a "hands-on" YAML file.